data/data-pipeline/data_pipeline/tests/sources/national_risk_index/data/output.csv

```diff
@@ -1,6 +1,3 @@
 GEOID10_TRACT,FEMA Risk Index Expected Annual Loss Score,Expected population loss rate (Natural Hazards Risk Index),Expected agricultural loss rate (Natural Hazards Risk Index),Expected building loss rate (Natural Hazards Risk Index)
-05007040300,11.5,0.241594948,0.206607506,0.399500316
-05001020100,12.5,0.281343259,0.207119742,0.535089827
-15007040500,13.5,0.280726185,0.269219337,0.493093767
-15001021010,14.5,0.221568983,0.360886597,0.305051347
-15001021101,15.5,0.805488269,0.204161204,0.30618612
+15007040300,18.619940187472693,3.5067096117382227e-06,0.0087782351557864,7.19506190745842e-05

```

Comment:
Huh, why are these in scientific notation?

Reply:
Good question - I thought that since this was just an artifact of numerical instability after the division operation. I did not see setting the numpy floating point limit or ceiling. We could coerce like this `math.ceil((some_ridiculous number * 1e15)/1e15)`, but that just looks cheap to me.

Reply:
I think we can just fix the `to_csv` parameters: https://stackoverflow.com/a/23006399/1569221

Reply:
This is still posing problems. We do this as well to standardize: 

`df_data = df_data.round(3)`
`df_data.to_csv('somename.csv')`

Reply:
I should have added this as well.

I added this to check why the assertion call keeps failing.

```
        difference_locations = np.where(etl.df != expected)
        changed_from = etl.df.values[difference_locations]
        changed_to = expected.values[difference_locations]
        df_2 = pd.DataFrame({'from': changed_from, 'to': changed_to}, index=np.arange(len(changed_to)))
        print(df_2)
```

The `stdout` is attached below. Notice that visually the differences are not apparent to the human eye using the suggestion provided in that post shared above. The load test is just fine by the way, which is why I am at a complete loss as to how to "fix" the test. That's what is blocking me. 

<img width="726" alt="Screen Shot 2021-12-03 at 7 10 34 AM" src="https://user-images.githubusercontent.com/7760225/144600764-e34a550c-6010-4ffb-8c02-ae4b9f725c8b.png">